src/List.css

```diff
@@ -0,0 +1,37 @@
+.container {
```

Comment:
FYI, the only down side of using a css file like this, is that it affects every part of your code and files. So, even if you import this css file inside `List.tsx` file, if you give container class to some other element in another file, it will inherit the properties of this container class. For bigger projects, this might be problematic.